openlibrary/templates/type/author/view.html

```diff
@@ -64,20 +64,29 @@
 
 $set_share_links(url=request.canonical_url, title=title, view_context=ctx)
 
+$if page.birth_date or page.death_date:
+    $ entityTypeSchema = "https://schema.org/Person"
+$elif page.entity_type:
+    $ entityTypeSchema = "https://schema.org/Organization"
+
 <div id="contentHead">
 
     $:macros.databarView(page)
 
-    <h1>$title</h1>
-    <h2 class="author collapse">
-    $if page.birth_date or page.death_date:
-        $page.birth_date - $page.death_date
-    $else:
-        $if page.date:
-            $page.date
-    </h2>
+    <div itemscope itemtype=$entityTypeSchema>
```

Comment:
We need this to go higher up still, so that there is only one `Person` on this page, and all the `itemprop`s apply to it. It looks like we need this to go onto the `body`.

Something like this should do it:

```
$ bodyattrs = ctx.setdefault('bodyattrs', [])
$ bodyattrs.append('itemscope')
$ bodyattrs.append('itemtype="%s"' % entityTypeSchema)
```

Then in `body.html`, add:

```
$ bodyattrs = ctx.get('bodyattrs', [])
<body id="$bodyid" class="$(' '.join(bodyclass))" $(' '.join(bodyattrs))> 
```

(I.e. do the equivalent of what was done here: https://github.com/internetarchive/openlibrary/commit/6c5dabbe9bd4e54b2b1a7e629a25369b05c863e3 and here: https://github.com/internetarchive/openlibrary/commit/906541efe925f2159d7254a690b63d597fc0657f )